src/view.cpp

```diff
@@ -359,8 +359,14 @@ void view::open_in_browser(const std::string& url) {
 		cmdline.append("'");
 	}
 	stfl::reset();
-	utils::run_interactively(cmdline, "view::open_in_browser");
+	int browser_exit_code = utils::run_interactively(cmdline, "view::open_in_browser");
+	if (browser_exit_code == 0) {
+		LOG(level::DEBUG, "Starting the browser failed with error code %d - command was %s\n", browser_exit_code, cmdline);
+	}
 	pop_current_formaction();
+
+	if (browser_exit_code) return false;
```

Comment:
You missed my previous comment on this: `return browser_exit_code == 0` is easier to understand. Just imagine the thought process of someone who will read the current version:

1. Ok, so `browser_exit_code` is an `int`, and it'll be converted to `bool` here.
2. So if it's non-zero, the condition evaluates to `true`, we're going to the first branch and function returns `false`;
3. ...And if it's zero, the condition is `false` and we return `true`.
4. Ah, so it just checks if the exit code is zero.

If you write `return browser_exit_code == 0`, you're essentially eliminating the first three steps above and jump right to the conclusion.